CHANGELOG.md

```diff
@@ -73,36 +73,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- [#5](https://github.com/green-code-initiative/ecoCode-python/pull/5) Upgrade licence system and licence headers of
+- [#5](https://github.com/green-code-initiative/creedengo-python/pull/5) Upgrade licence system and licence headers of
   Java files
-- [#6](https://github.com/green-code-initiative/ecoCode-python/pull/6) Adding EC35 rule : EC35 rule replaces EC34 with a
+- [#6](https://github.com/green-code-initiative/creedengo-python/pull/6) Adding EC35 rule : EC35 rule replaces EC34 with a
   specific use case ("file not found" specific)
-- [#7](https://github.com/green-code-initiative/ecoCode-python/issues/7) Add build number to manifest
-- [#123](https://github.com/green-code-initiative/ecoCode/issues/123) Improve unit tests for EC7 rule
-- Update ecocode-rules-specifications to 1.4.6
+- [#7](https://github.com/green-code-initiative/creedengo-python/issues/7) Add build number to manifest
+- [#123](https://github.com/green-code-initiative/creedengo/issues/123) Improve unit tests for EC7 rule
+- Update creedengo-rules-specifications to 1.4.6
```

Comment:
I'm ok to rename "ecocode" to "creedengo" inside all links but not Ok to rename old descriptions on CHANGELOG.md : for me the historical descirption have to be kept ... same feedback for all description changes in this file